subset/switches/src/main/java/switchtest/SwitchInterrogator.java

```diff
@@ -452,6 +505,10 @@ private void validateTests() {
       String current_duplex = show_interface_data[3];
       String configured_duplex = show_interface_data[6];
 
+      String current_max_power = show_power_data[7];
+      String current_power = show_power_data[4];
```

Comment:
just for future reference, doing this with all these embedded constants is horribly. please try to never do it again. You can implement as an enum next time, so you can have each entry both be by named key and value.